### PR TITLE
Feature/error handling tests

### PR DESCRIPTION
**Pull request by MichaelDally**
Error handling tests added for URL routes

### CODE REVIEW

File: ApplyToBecomeInternal/ApplyToBecomeInternal/wwwroot/package-lock.json [hunk 1]
```diff
@@ -1,6 +1,2180 @@
 {
+	"name": "wwwroot",
```
Comment (danielburnley) [inline ApplyToBecomeInternal/ApplyToBecomeInternal/wwwroot/package-lock.json:2]: 👀 we should revert this unless we all decide to move to the latest version of node, which we should sort out at some point

File: ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/error-handling/incorrect-url-tests.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+		);
+	});
+
+    // Raised under 80466
+    it.skip('Should display user-friendly error when incorrect return parameter passed [80466]',()=>{
+        let modifiedUrl
+        cy.get('[aria-describedby*=la-info-template-status]').click()
+        cy.url().then(url =>{
+            //Changes the current URL to:
+            //https://apply-to-become-an-academy-internal-dev.london.cloudapps.digital/task-list/<SOME_VALID_ID>/confirm-local-authority-information-template-dates?return=someInvalideParam/SomeInvalidPath
```
Comment (danielburnley) [inline ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/error-handling/incorrect-url-tests.js:27]:
```suggestion
            ///task-list/<SOME_VALID_ID>/confirm-local-authority-information-template-dates?return=someInvalideParam/SomeInvalidPath
```

removing mentions to specific urls

**Review by danielburnley — APPROVED** [review]
Just a couple smol comments, feel free to merge when youve addressed them 🎉 just trying to avoid too much churn back and forth on the package lock till we can all move to the latest version

**Comment by MichaelDally** [comment]
In case this breaks something I ran:

```git checkout main -- package-lock.json``` inside the wwroot folder

**Pull request merged**